v2/devdoc/logging.md

```diff
@@ -0,0 +1,339 @@
+# `c-logging` requirements
+
+## Basic functionality
+
+`c-logging` shall expose APIs allowing the user to log events that can be used for observing the software activity, debugging, etc.
+
+The exposed API shall allow creating contexts both dynamically and on the stack and using those contexts for logging.
+
+The API shall allow easy formatting of values to be placed in the contexts by using printf-like formatting.
+
+```
```

Comment:
we have a chapter named "Exposed API".

Missing "Exposed api"

Reply:
Added an Exposed API.